CFC_WebApp/main/Trip.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import division
+from dao.user import User
+from googlemaps import GoogleMaps
+from pymongo import MongoClient
+import urllib2
+import xml.etree.ElementTree as ET
+API_KEYS = ("AIzaSyBBcTJ5g9sLrufvAE5BC_KhYZ5ecmbOUxU")
```

Comment:
Don't check in API keys into source code. This really needs to be in a separate config file.